src/Controls/src/Core/TrimmerConstants.cs

```diff
@@ -8,4 +8,7 @@ class TrimmerConstants
 	internal const string NativeBindingService = "This method properly handles missing properties, and there is not a way to preserve them from this method.";
 
 	internal const string XamlRuntimeParsingNotSupportedWarning = "Loading XAML at runtime might require types and members that cannot be statically analyzed. Make sure all of the required types and members are preserved.";
+
+	internal const string QueryPropertyAttributeWarning = "Using QueryPropertyAttribute is not trimming friendly and might not work correctly. Implement the IQueryAttributable interface instead.";
+	internal const string QueryPropertyDocsUrl = "https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/shell/navigation#process-navigation-data-using-a-single-method";
```

Comment:
Oh actually we should remove `en-us` from URLs:
```suggestion
	internal const string QueryPropertyDocsUrl = "https://learn.microsoft.com/dotnet/maui/fundamentals/shell/navigation#process-navigation-data-using-a-single-method";
```
MS docs can select language appropriately.